pkg/runtime/service_controller_test.go

```diff
@@ -99,13 +99,7 @@ func TestServiceController(t *testing.T) {
 	reg := ackrt.NewRegistry()
 	reg.RegisterResourceManagerFactory(rmf)
 
-	vi := ackrt.VersionInfo{
-		GitCommit:  "test-commit",
-		GitVersion: "test-version",
-		BuildDate:  "now",
-	}
-
-	sc := ackrt.NewServiceController("bookstore", "bookstore.services.k8s.aws", vi)
```

Comment:
you just need to rebase with the latest runtime repo code. The above is necessary due to the breakout of the runtime and service controller code and not having a single pkg/version to use.